### PR TITLE
Arbitrage tutorial contract

### PR DESCRIPTION
**Pull request by LogvinovLeon**
## Types of changes

*   [ ] Bug fix (non-breaking change which fixes an issue)
*   [ ] New feature (non-breaking change which adds functionality)
*   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

*   [ ] Change requires a change to the documentation.
*   [ ] Added tests to cover my changes.
*   [ ] Added new entries to the relevant CHANGELOGs.
*   [ ] Updated the new versions of the changed packages in the relevant CHANGELOGs.
*   [ ] Labeled this PR with the 'WIP' label if it is a work in progress.
*   [ ] Labeled this PR with the labels corresponding to the changed package.

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
     "lerna": "2.5.1",
     "packages": ["packages/*"],
+    "commands": {
+        "publish": {
+            "allowBranch": "development"
+        }
+    },
```
Comment (fabioberger) [inline lerna.json]: Why was this added?

Comment (LogvinovLeon) [reply lerna.json]: This prevents us from accidentally publishing from another branch. Just a safety check
